ls_handleTouches.ino

```diff
@@ -1732,18 +1732,23 @@ void handleTouchRelease() {
       preSendLoudness(sensorSplit, 0, 0, sensorCell->note, sensorCell->channel);
     }
 
-    // unregister the note <> cell mapping
-    if (!isSwitchLegatoPressed(sensorSplit) && (!isArpeggiatorEnabled(sensorSplit) || !isSwitchLatchPressed(sensorSplit))) {
-      noteTouchMapping[sensorSplit].noteOff(sensorCell->note, sensorCell->channel);
```

Comment:
For the arpeggiator fix, the main idea is to prevent the immediate calling of noteOff events.
So the deliberate change here is to prevent calling `noteTouchMapping[sensorSplit].noteOff` if we're arping.